docs/src/examples/components/Alert/BestPractices/AlertBestPractices.tsx

```diff
@@ -0,0 +1,15 @@
+import * as React from 'react'
+
+import ComponentBestPractices from 'docs/src/components/ComponentBestPractices'
+
+const doList = [
+  'Use warning and danger variants to announce the alert by the screen reader.',
+  'Use other libraries (for example `react-aria-live`) if the content of default or success variant needs to be announced.',
```

Comment:
unfortunately, these markdown ticks are rendered 'as is' at the moment - however, think that this is a right direction taken, and it is just about providing simple markdown renderer to process these lines (like `react-markdown`). Otherwise, if we will use React markup in `do` and `don't` for these simple `code` items, we will end up with barely readable code. Lets just take this as a follow-up thing